Content.Shared/Blocking/Components/SharedBlockingUserComponent.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Content.Shared.Damage;
+using Robust.Shared.Physics;
+
+namespace Content.Shared.Blocking;
+
+/// <summary>
+/// This component gets dynamically added to an Entity via the <see cref="SharedBlockingSystem"/>
+/// </summary>
+[RegisterComponent]
+public class SharedBlockingUserComponent : Component
+{
+    /// <summary>
+    /// The entity that's being used to block
+    /// </summary>
+    public EntityUid? BlockingItem;
+
+    public DamageModifierSet Modifiers = default!;
```

Comment:
As above on datafields and also VV as it will be useful for debugging.

Reply:
Do they need datafields since this isn't used in yamls  but only via dynamically adding via the blocking system?

Reply:
Good to know, I thought it was purely for yml interaction.